docs/guides/migration-from-redux.md

```diff
@@ -0,0 +1,295 @@
+        return todos.filter(t => t.completed)
+      case 'SHOW_ACTIVE':
+        return todos.filter(t => !t.completed)
+    }
+  }
+)
+```
+
+**Reatom**
+```js
+import { map, combine } from '@reatom/core'
+
+export const visibleTodosAtom = map(combine([
+  visibilityFilterAtom, todosAtom
+]), ([visibilityFilter, todos]) => {
+  switch (visibilityFilter) {
+    case 'SHOW_ALL':
+      return todos
+    case 'SHOW_COMPLETED':
+      return todos.filter(t => t.completed)
+    case 'SHOW_ACTIVE':
+      return todos.filter(t => !t.completed)
+  }
+})
+```
+
+## Creating store
+Creating a store is similar to how you did it in Redux.
+
+**Redux**
+```js
+import { createStore } from 'redux'
+
+const store = createStore(rootReducer, preloadedData)
+```
+
+**Reatom**
+```js
+import { createStore } from '@reatom/core'
+
+const store = createStore(rootAtom, preloadedData)
+```
+
+> NOTE. All arguments is optional. You can create store like  `createStore()`
+
+## Getting state
+Getting state is similar to how you did it in Redux, but with small differences.
+
+**Redux**
+```js
+store.getState()
+```
+
+**Reatom**
+```js
+// Get full store state 
+const storeState = store.getState()
+
+// Get atom state
+const atomState = store.getState(myAtom)
+```
+> NOTE. Use full store state only if needs, because this affects performance. In most cases you need to use atom state.
+
+## Subscribing
+Subscribing to the store is similar to how you did it in Redux, but with small differences.
+
+**Redux**
+```js
+import { createSelector } from 'reselect'
+
+const onVisibleTodosUpdate = createSelector(
+  getVisibleTodos,
+  todosState => {
+    // ...do something
+  }
+)
+
+const unsubscribe = store.subscribe(
+  () => onVisibleTodosUpdate(store.getState())
+)
+
+unsubscribe()
+```
+
+**Reatom**
+
+Subscribing to the actions
+```js
+const unsubscribe = store.subscribe(action => {
+  // Always called after dispatch (but after updating atoms)
+  // ...do something
+})
+
+unsubscribe()
+```
+Subscribing to the atoms
+```js
+const unsubscribe = store.subscribe(visisbleTodosAtom, todosState => {
+  // Will be called after update atom state
+  // ...do something
+})
+
+unsubscribe()
+```
+
+## Dispatching
+Dispatching to the store is similar to how you did it in Redux.
+
+**Redux**
+```js
+// via action creator
+store.dispatch(addTodo('Hello Redux'))
+
+// via object notation
+store.dispatch({
+  type: 'ADD_TODO',
+  payload: 'Hello Redux'
+})
+```
+
+**Reatom**
+```js
+// via declarated action
+store.dispatch(addTodo('Hello Reatom'))
+
+// via object notation (if needs)
+store.dispatch({
+  type: 'ADD_TODO',
+  payload: 'Hello Reatom'
+})
+```
+
+## Async actions
+In Redux world your needs use special libraries ([redux-thunk](github.com/reduxjs/redux-thunk) or [redux-saga](https://github.com/redux-saga/redux-saga)) for creating async actions. In Reatom world your don't needs use it. This role is played by [reactions](/glossary?id=action-reactions).
```

Comment:
Possibly "In Redux world **you need to** use special libraries"

and

"In Reatom world **you don't need to** use it."